ludwig/schema/features/utils.py

```diff
@@ -35,8 +35,18 @@ def update_encoders(feature_props, feature_type):
         None
     """
     for key in get_encoder_classes(feature_type):
-        if key not in feature_props["encoder"]["enum"]:
-            feature_props["encoder"]["enum"].append(key)
+        encoder_props = feature_props["encoder"]
```

Comment:
This looks non-trivial. Can we add a unit test for `update_encoders`?

Reply:
Will do in a separate PR: https://linear.app/predibase/issue/MLQ-38